queue/queue.go

```diff
@@ -101,8 +109,18 @@ func (q *queue) initialize() error {
 }
 
 func (q *queue) Publish(ctx context.Context, msg *Message) error {
+	var txn *profiler.Tracker
+	txn, ctx = q.profiler.StartQueueInteractionTracking(ctx, "publish_"+q.streamName+"_"+q.subject)
```

Comment:
let's use - instead of _, current practice in system
```suggestion
	txn, ctx = q.profiler.StartQueueInteractionTracking(ctx, "publish-"+q.streamName+"-"+q.subject)
```